static/app/views/monitors/components/checkInTimeline.tsx

```diff
@@ -38,35 +30,43 @@ function getBucketedCheckInsPosition(
 }
 
 export function CheckInTimeline(props: Props) {
-  const {bucketedData, start, end} = props;
+  const {bucketedData, start, end, timeWindow} = props;
 
   function renderTimelineWithWidth(width: number) {
-    const timeWindow = end.getTime() - start.getTime();
-    const msPerPixel = timeWindow / width;
+    const elapsedMs = end.getTime() - start.getTime();
+    const msPerPixel = elapsedMs / width;
 
     const jobTicks = mergeBuckets(bucketedData);
 
     return (
       <TimelineContainer>
-        {jobTicks.map(
-          ({startTs, width: tickWidth, envMapping, roundedLeft, roundedRight}) => {
-            const timestampMs = startTs * 1000;
-            const left = getBucketedCheckInsPosition(timestampMs, start, msPerPixel);
+        {jobTicks.map(jobTick => {
+          const {
+            startTs,
+            width: tickWidth,
+            envMapping,
+            roundedLeft,
+            roundedRight,
+          } = jobTick;
```

Comment:
non blocking but you could inline this
```suggestion
        {jobTicks.map(({
            startTs,
            width: tickWidth,
            envMapping,
            roundedLeft,
            roundedRight,
          }) => {;
```

Reply:
ooh i think i specifically didn't inline it just so i could easily pass `jobTick` down to the tooltip

Reply:
ohhh I missed that!